CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# Unreleased
+
+* Add support for http over unix sockets - requires hackey >= 1.6.2, erlang >= 19. (#185).
```

Comment:
There is a typo: `hackey` → `hackney`.
Also you probably should bump the hackney's version to 1.6.2 in [`mix.exs`](https://github.com/edgurgel/httpoison/blob/master/mix.exs) if this is the version in which the feature was implemented.